pom.xml

```
@@ -200,7 +200,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>2.13.0</version>
+        <version>2.14.0-rc1</version>
```

2.14.1 is out now, prefer to not depend on rc version

```suggestion
        <version>2.14.1</version>
```